changelog/unreleased/issue-4737

```diff
@@ -0,0 +1,3 @@
+Enhancement: include snapshot id in reason field of forget JSON output
+
+The JSON output of the `forget` command now includes the `id` and `short_id` of a snapshot in the `reason` field.
```

Comment:
Please add a link to this PR as the last line of the changelog:
```
https://github.com/restic/restic/pull/4737
```

Reply:
Link added